docs/development/extensions-core/lookups-cached-global.md

```diff
@@ -352,6 +352,7 @@ The JDBC lookups will poll a database to populate its local cache. If the `tsCol
 |`filter`|The filter to use when selecting lookups, this is used to create a where clause on lookup population|No|No Filter|
 |`tsColumn`| The column in `table` which contains when the key was updated|No|Not used|
 |`pollPeriod`|How often to poll the DB|No|0 (only once)|
+|`jitterSeconds`| How much jitter to add (in seconds) as a delay, used to distribute db load more evenly|No|0
```

Comment:
table formatting missing ending `|`
```suggestion
|`jitterSeconds`| How much jitter to add (in seconds) as a delay, used to distribute db load more evenly|No|0|
```

Reply:
also should we indicate that the actual value used will be random up to this maximum?